### ISSUE TITLE
Is it possible to send the reporter multiple json files?

### ISSUE THREAD

**Issue by sgupta304** [gkushang/cucumber-html-reporter#149]
I have a suite that runs tests in parallel, and each creates its own json file. Is it possible to send multiple json files, and create a single html report from it?

**Comment by gkushang** [comment]
if you can get all the JSON reports in a single directory then you can set [jsonDir](https://github.com/gkushang/cucumber-html-reporter#jsondir) instead of `jsonFile` while sending reports to the Reporter. It would consolidate and generate HTML.

**Comment by sgupta304** [comment]
Yes, they are all generated in a single directory. I'll try that out! Thanks so much!

**Comment by sgupta304** [comment]
![image](https://user-images.githubusercontent.com/9002815/42538759-6e4aa408-845e-11e8-8310-cfce180bb2ad.png)

My Pie Charts are not showing correctly :(

**Comment by sgupta304** [comment]
![image](https://user-images.githubusercontent.com/9002815/42538881-d508ca1c-845e-11e8-8012-06f9389bc539.png)

Actually my entire report is not showing up correct. Anything I'm missing?

**Comment by sgupta304** [comment]
@gkushang Can you please help out with the chart visualization. Once it is working properly I can close out the ticket. Thanks

**Comment by gkushang** [comment]
Nice. Pls, let me know what other issues you are facing?

**Comment by sgupta304** [comment]
@gkushang Just the pie charts are not showing up correctly. They have a white band in the middle.

**Comment by gkushang** [comment]
@sgupta304 that is by design. The white band means the tests are broken (as like shown in pie-chart). the white band wouldn't appear if all tests are green (green pie-chart)
